water-alternatives.csl

```diff
@@ -45,19 +45,31 @@
           </names>
           <group delimiter=" ">
             <choose>
-              <if variable="ISBN ISSN" match="any">
+              <if variable="ISBN ISSN container-title" match="any">
```

Comment:
No, this construction is due to different requirements for conference paper published in official proceedings book and unpublished articles. I suppose the proceeding have the name or ISBN or ISSN. I add "container-title" to the condition because in some cases the conference proceedings book has not ISSN or ISBN. Unfortunately, in my field of focus (Water management), it is very often in the Czech Republic.